benchmarks/benchmarks/stats.py

```diff
@@ -174,3 +184,51 @@ def time_binned_statistic_dd_reuse_bin(self, statistic):
         stats.binned_statistic_dd(
             [self.inp[0], self.inp[1]], self.inp[2], statistic=statistic,
             binned_statistic_result=self.ret)
+
+
+class ContinuousFitAnalyticalMLEOverride(Benchmark):
+    # list of distributions to time
+    dists = ["pareto", "laplace", "rayleigh", "invgauss"]
+    # add custom values for rvs and fit, if desired, for any distribution:
+    # key should match name in dists and value should be list of loc, scale,
+    # and shapes
+    custom_input = {}
+    fnames = ['floc', 'fscale', 'f0', 'f1', 'f2']
+    fixed = {}
+    distcont = dict(distcont)
+
+    param_names = ["distribution", "loc_fixed", "scale_fixed",
+                   "shape1_fixed", "shape2_fixed", "shape3_fixed"]
+    params = [dists, * [[True, False]] * 5]
+
+    def setup(self, dist_name, loc_fixed, scale_fixed, shape1_fixed,
+              shape2_fixed, shape3_fixed):
+        self.distn = eval("stats." + dist_name)
```

Comment:
Maybe change this to `getattr(stats, distname)` while you're at it. It's a little cleaner.